crypto/internal.h

```diff
@@ -1021,7 +1022,19 @@ OPENSSL_INLINE int boringssl_fips_break_test(const char *test) {
 //   5: vpaes_set_encrypt_key
 //   6: sha256_block_data_order_shaext
 //   7: aes_gcm_encrypt_avx512
+// On AARCH64:
+//   0: aes_hw_ctr32_encrypt_blocks
+//   1: aes_hw_encrypt
+//   2: aes_gcm_enc_kernel
+//   3: aes_hw_set_encrypt_key
+//   4: vpaes_encrypt
+//   5: vpaes_set_encrypt_key
+//   6: aesv8_gcm_8x_enc_128
+#if defined(OPENSSL_AARCH64)
```

Comment:
Does this need to match the change made in cpucap.c to keep the larger array only?

Reply:
I must have missed this when removing the larger array, although it seems like it isn't causing a problem with the code working I'll remove it anyway so that the code will be cleaner / have fewer ifdefs :)